### PR TITLE
Add base methods

### PR DESCRIPTION
**Pull request by otseobande**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  static distinct () {
+    throw new NotImplementedError('distinct method not implemented');
+  }
+
+  static delete () {
+    throw new NotImplementedError('delete method not implemented');
+  }
+
+  static drop () {
```
Comment (nawazdhandala) [inline src/index.js]: remove this.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    throw new NotImplementedError('drop method not implemented');
+  }
+
+  static dropDatabase () {
```
Comment (nawazdhandala) [inline src/index.js]: deleteDatabase

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    throw new NotImplementedError('dropDatabase method not implemented');
+  }
+
+  static create () {
```
Comment (nawazdhandala) [inline src/index.js]: What does this create?

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+  static getIndexes () {
+    throw new NotImplementedError('getIndexes method not implemented');
+  }
+
+  static renameColumn () {
+    throw new NotImplementedError('renameColumn method not implemented');
+  }
+
+  static dropColumn () {
```
Comment (nawazdhandala) [inline src/index.js]: deleteColumn

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    throw new NotImplementedError('dropColumn method not implemented');
+  }
+
+  static list () {
```
Comment (nawazdhandala) [inline src/index.js]: What does this do?

Comment (otseobande) [reply src/index.js]: In mongo it returns all entries in the `_Schema` collection

File: src/index.js (hunk 3 above)
Comment (otseobande) [reply src/index.js]: In mongo it creates and returns a replSet db instance

File: src/index.js (hunk 1 above)
Comment (otseobande) [reply src/index.js]: Okay

File: src/index.js (hunk 2 above)
Comment (otseobande) [reply src/index.js]: Noted

File: src/index.js (hunk 4 above)
Comment (otseobande) [reply src/index.js]: Noted

File: src/index.js (hunk 5 above)
Comment (otseobande) [reply src/index.js]: Should I remove this?

File: src/index.js (hunk 3 above)
Comment (otseobande) [reply src/index.js]: Should I remove it?

**Pull request merged**